src/algorithms/drls.jl

```diff
@@ -53,7 +53,7 @@ Base.@kwdef mutable struct DRLSState{R,Tx,TH}
     res::Tx
     res_prev::Tx = similar(x)
     xbar::Tx
-    xbar_prev::Tx = similar(x)
+    xbar_prev::Tx = copy(xbar)
```

Comment:
@nantonel This was one of the bugs you were afraid of, tests caught it (but didn't before!) 🙂

Reply:
eheh! told you `similar` can be dangerous!